types/enquire.js/enquire.js-tests.ts

```diff
@@ -0,0 +1,17 @@
+import * as enquire from 'enquire.js';
```

Comment:
Please use `import enquire = require('enquire.js')` to match the `export = enquire` syntax.

Reply:
Fixed @sandersn